README.md

```diff
@@ -22,28 +87,20 @@ via `getRecord()` method. Will return a number, if number is gte max, return
 
 ## ThrottlerStorage
 
-Class to handle the details when it comes to keeping track of the requests.
-Early implementation would be something like this
+Interface to define the methods to handle the details when it comes to keeping track of the requests.
 
-```ts
-class ThrottlerStorage {
-  private storage: Record<string, number>;
+Currently the key is seen as an `MD5` hash of the `IP` the `ClassName` and the `MethodName`, to ensure that no unsafe characters are used and to ensure that the package works for contexts that don't have explicit routes (like Websockets and GraphQL).
 
-  addRecord(key: string, ttl: number): void {
-    this.storage[key] = this.storage[key] ? this.storage[key] + 1 : 1;
-    setTimeout(() => this.storage[key]--, ttl * 1000)
-  }
+The interface looks like this:
 
-  getRecord(key: string): number {
-    return this.storage[key] || 0;
-  }
+```ts
+export interface ThrottlerStorage {
+  getRecord(key: string): Date[];
```

Comment:
As we discussed, we changed the interface `Date[]` to be `number[]`. Make sure to change this.

Reply:
Change comes in the next patch, right? Currently this branch still uses `Date[]`